src/App.css

```
@@ -127,87 +161,107 @@ nav.navbar{
-    box-shadow: 20px 20px 10px grey;
+    text-align: center; 
+    background-color: var(--surface);
+    border-radius: 10px;
+    box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
 }
 
 .StatusArea {
     padding-top: 7px;
-    height: 120px;
-    width: 280px;
+    padding: 15px;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-top



(lint/suspicious/noShorthandPropertyOverrides)